crosswalks/spase-v2.6.1.csv

```diff
@@ -0,0 +1,74 @@
+Property,SPASE
+codeRepository,AccessInformation/RepositoryID
+programmingLanguage,codeLanguage
+review,
+runtimePlatform,
+targetProduct,
+applicationCategory,
+applicationSubCategory,
+downloadUrl,AccessURL/URL
+fileSize,
+installUrl,ExecutionEnvironment/Installer/URL
+memoryRequirements,ExecutionEnvironment/Memory
+operatingSystem,ExecutionEnvironment/OperatingSystems
+permissions,
+processorRequirements,
+releaseNotes,
+softwareHelp,
+softwareRequirements,Prerequisites
+softwareVersion,SoftwareVersion
+storageRequirements,ExecutionEnvironment/Storage
+supportingData,
+author,
+citation,
+contributor,
+copyrightHolder,
+copyrightYear,
+dateCreated,
+dateModified,
+datePublished,ResourceHeader/ReleaseDate
+editor,
+encoding,AccessInformation/Encoding
+fileFormat,AccessInformation/Format
+funder,ResourceHeader/Funding/Agency
+keywords,
+license,
+producer,
+provider,
+publisher,
+sponsor,
+version,
+isAccessibleForFree,
+isPartOf,
+hasPart,
+position,
+description,ResourceHeader/Description
+identifier,ResourceID
+name,ResourceHeader/ResourceName
+sameAs,
+url,ResourceHeader/InformationURL/URL
+relatedLink,
+givenName,
+familyName,
+email,
+affiliation,
+identifier,ResourceHeader/DOI
+name,
+address,
+reviewAspect,
+reviewBody,
+endDate,
+roleName,
+startDate,
+softwareSuggestions,
+maintainer,ResourceHeader/Contact
+continuousIntegration,
+buildInstructions,
+developmentStatus,
+embargoEndDate,
+funding,ResourceHeader/Funding/AwardNumber or ResourceHeader/Funding/Project (if no AwardNumber)
```

Comment:
```suggestion
funding,ResourceHeader/Funding/AwardNumber or ResourceHeader/Funding/Project
```

Reply:
@BaptisteCecconi can you please tell us if you are ok with this suggestion, so @progval can merge the PR?